### ISSUE TITLE
Temporarily changing the theme for certain pages

### ISSUE THREAD

**Issue by achadee** [pacocoursey/next-themes#42]
Hi there, firstly, I love this project and thanks for working on it!

We have a few custom landing pages where the theme is dark. I was wondering if there was an option (or a way) to temporarily set the theme just for that page and not store the the new theme in localStorage. That way when the user visits another page, it doesn't carry across the theme.

Thanks

**Comment by pacocoursey** [comment]
Yep, that's exactly what the `forcedTheme` option does. Set a static property on your Page:

https://github.com/pacocoursey/next-themes/blob/cb30afc83a80eab7eca66f985f23ddb01a14a2eb/examples/example/pages/dark.js#L11

Then pass it along in your `_app`:

https://github.com/pacocoursey/next-themes/blob/cb30afc83a80eab7eca66f985f23ddb01a14a2eb/examples/example/pages/_app.js#L6

It will set the theme for the page without modifying localStorage.

**Comment by achadee** [comment]
Beautiful! thank you so much, works a treat :)
